### DISCUSSION TITLE
Coolify Integration

### DISCUSSION THREAD

**Discussion by ngregrichardson** [aptabase/aptabase#87] [General]
Hey! I've been really loving Aptabase, it's beautiful and exactly what I need. I love self-hosting stuff like this, and I use [Coolify](https://coolify.io/) for a lot of my stuff. They have one-click integrations for stuff like this, and I wanted to ask if the maintainers would be alright with me making one. The guide is [here](https://coolify.io/docs/knowledge-base/add-a-service) in case you're curious as to how it works, it's basically just adding a docker compose file to their repository along with a logo.

If you're alright with this and have a specific logo that you'd like (SVG, png, whatever might look best in light _and_ dark mode), please let me know!

Thanks for building this!

**Comment by JavascriptMick** [comment]
hey @ngregrichardson have you figured this out? I thought it might be worth a try to just add a docker compose resource to an existing project and specify the self hosting docker compose file https://github.com/aptabase/self-hosting/blob/main/docker-compose.yml would this work?

**Comment by awpogodin** [comment]
I’ve created a ready-to-use Docker Compose setup for deploying Aptabase on Coolify 🚀

✅ Includes PostgreSQL & ClickHouse  
🔐 Uses environment variables and health checks  
🌐 Works with Coolify's proxy + Let's Encrypt  
🔗 [aptabase-coolify](https://github.com/awpogodin/aptabase-coolify)

Perfect for anyone who wants to get Aptabase up and running in Coolify with minimal effort. Check it out!
